src/model.jl

```diff
@@ -263,10 +289,15 @@ function _add_constraint_load_shed!(
         bus_demand[sets.load_bus_idx[i], j],
     )
     if demand_flexibility.enabled
+        flexible_load_bus_idx = indexin(sets.flexible_bus_idx, sets.load_bus_idx)
+        flexible_load_map = sparse(
+                flexible_load_bus_idx, 1:sets.num_flexible_bus, 1, sets.num_load_bus, sets.num_flexible_bus
+            )::SparseMatrixCSC
```

Comment:
Is this duplicated from the calculation in `_make_sets`? Within the expression definition below, can we use `sets.flexible_load_bus_map` instead?

Reply:
Ah, good catch.